### DISCUSSION TITLE
Edges not cutting Well

### DISCUSSION THREAD

**Discussion by BC314** [arkypita/LaserGRBL#1418] [Q&A]
I'm using my laser engraver for simple shapes, but the sharp edges aren't cutting very well. Is there a good way to fix this problem?

![IMG_4454](https://user-images.githubusercontent.com/35663754/113530552-2f2e1f80-957b-11eb-8b8d-6b9300e79f6e.JPG)

**Comment by arkypita** [comment]
Use M3 instead of M4.
Run slower.
